extension/package.json

```diff
@@ -54,6 +54,22 @@
         "group": "navigation"
       }
     ],
+    "configuration": {
+      "type": "object",
+      "title": "XComponent Configuration",
+      "properties": {
+        "xcbuild": {
+					"type": "string",
+					"default": "",
+					"description": "Path to xcBuild"
+        },
+        "mono": {
+					"type": "string",
+					"default": "",
```

Comment:
Same comment.
the default value should be mono
(because mono is the path env by default on osx/linux when installed)

Reply:
We should rename this part and use mono.facades.path (because it is really what you want)